nix/haskell-pins.nix

```diff
@@ -172,6 +172,13 @@ let
         sha256 = "1w23yz2iiayniymk7k4g8gww7268187cayw0c8m3bz2hbnvbyfbc";
       };
     };
+    swagger2 = {
```

Comment:
Why this Swagger2 pin? I suppose it provides something that currently doesn't exist in the version that is used.